tests/baselines/reference/fuzzy.errors.txt

```diff
@@ -39,7 +38,6 @@ tests/cases/compiler/fuzzy.ts(25,20): error TS2352: Type '{ oneI: this; }' canno
 !!! error TS2322:   Types of property 'oneI' are incompatible.
 !!! error TS2322:     Type 'this' is not assignable to type 'I'.
 !!! error TS2322:       Type 'C' is not assignable to type 'I'.
-!!! error TS2322:         Property 'alsoWorks' is missing in type 'C'.

```

Comment:
This elaboration disappears as the comparison is now cached correctly and is the same comparison done above for the implements constraint check.